content/bn/_index.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "ক্লাউড নেটিভ শব্দকোষ"
+---
+
+# ক্লাউড নেটিভ শব্দকোষ
+
+ক্লাউড নেটিভ শব্দকোষ হল CNCF বিজনেস ভ্যালু সাবকমিটি (BVS) এর নেতৃত্বে একটি প্রকল্প। এর লক্ষ্য হল ক্লাউড নেটিভ ধারণাগুলিকে পরিষ্কার এবং সহজ ভাষায় ব্যাখ্যা করা কোনো পূর্বের প্রযুক্তিগত জ্ঞানের প্রয়োজন ছাড়াই।[আপনি এখানে (ইংরেজিতে) PDF সংস্করণ দেখতে বা ডাউনলোড করতে পারেন।](https://github.com/cncf/glossary/blob/main/cloudnative-glossary.pdf)
+
+## অবদান
+
+ক্লাউড নেটিভ শব্দকোষে পরিবর্তন, সংযোজন এবং উন্নতির পরামর্শ দেওয়ার জন্য সবাইকে আমন্ত্রণ জানানো হয়েছে। আমরা এই ভাগ করা অভিধানের বিকাশ এবং উন্নতির জন্য CNCF দ্বারা পরিচালিত একটি সম্প্রদায়-চালিত প্রক্রিয়া নিযুক্ত করি। এই শব্দকোষটি ক্লাউড নেটিভ প্রযুক্তির আশেপাশে একটি ভাগ করা শব্দভাণ্ডার সংগঠিত করার জন্য একটি বিক্রেতা-নিরপেক্ষ প্ল্যাটফর্ম প্রদান করে৷ প্রজেক্টের উদ্দেশ্য এবং চার্টার মেনে চলা সকল অংশগ্রহণকারীদের থেকে অবদানকে স্বাগত জানানো হয়।
+
+যে কেউ একটি অবদান করতে ইচ্ছুক একটি GitHub সমস্যা(issue) জমা দিতে বা একটি পুল অনুরোধ (pull request) তৈরি করতে পারেন. অনুগ্রহ করে নিশ্চিত করুন যে আপনি [শৈলী নির্দেশিকা](/bn/style-guide/) অনুসরণ করছেন, [কীভাবে অবদান রাখবেন](/bn/contribute/) ডকটি পড়ুন এবং CNCF স্ল্যাকের #glossary চ্যানেলে যোগদান করুন। এছাড়াও যারা তাদের মাতৃভাষায় শব্দকোষ অনুবাদ করতে সাহায্য করতে চান তাদের জন্য একটি #glossary-localizations চ্যানেল রয়েছে।
+
+## স্বীকৃতি
+
+ক্লাউড নেটিভ শব্দকোষটি CNCF মার্কেটিং দ্বারা সূচিত হয়েছিল
+কমিটি (CNCF Marketing Committee) এবং অন্তর্ভুক্ত
+থেকে অবদান [Catherine Paganini](https://www.linkedin.com/in/catherinepaganini/en/), [Chris Aniszczyk](https://www.linkedin.com/in/caniszczyk/),
+[Daniel Jones](https://www.linkedin.com/in/danieljoneseb/?originalSubdomain=uk), [Jason Morgan](https://www.linkedin.com/in/jasonmorgan2/), [Katelin Ramer](https://www.linkedin.com/in/katelinramer/), [Mike Foster](https://www.linkedin.com/in/mfosterche/?originalSubdomain=ca), [Seokho Son](https://www.linkedin.com/in/seokho-son/),[MD Shahriyar Al Mustakim Mitul](https://www.linkedin.com/in/md-shahriyar-al-mustakim-mitul-9084b31a0/), [Umme Abira Azmary](https://www.linkedin.com/in/umme-abira-azmary-68404a1bb/),[MD Ikramul Kayes](https://www.linkedin.com/in/md-ikramul-kayes-753674214/) এবং আরো অনেক অবদানকারী। একটি সম্পূর্ণ অবদানকারী তালিকার জন্য, অনুগ্রহ করে [এই GitHub পৃষ্ঠা](https://github.com/cncf/glossary/graphs/contributors) দেখুন.
```

Comment:
Please update this "Acknowledgements" section properly, according to the current English version,
while adding acknowledgements to initial Bengali localization contributors list properly too. 😊

[English version]
https://glossary.cncf.io/#acknowledgements
> The Cloud Native Glossary was initiated by the CNCF Marketing Committee (**Business Value Subcommittee**) and includes contributions from [Catherine Paganini](https://www.linkedin.com/in/catherinepaganini/en/), [Chris Aniszczyk](https://www.linkedin.com/in/caniszczyk/), [Daniel Jones](https://www.linkedin.com/in/danieljoneseb/?originalSubdomain=uk), [Jason Morgan](https://www.linkedin.com/in/jasonmorgan2/), [Katelin Ramer](https://www.linkedin.com/in/katelinramer/), [Mike Foster](https://www.linkedin.com/in/mfosterche/?originalSubdomain=ca), and many more contributors. For a complete contributor list, please refer to [this GitHub page](https://github.com/cncf/glossary/graphs/contributors).
> 
> The Glossary is maintained by [Catherine Paganini](https://www.linkedin.com/in/catherinepaganini/en/), [Jason Morgan](https://www.linkedin.com/in/jasonmorgan2/), [Jihoon Seo](https://www.linkedin.com/in/jihoon-seo/), [Noah Ispas](https://www.linkedin.com/in/noah-ispas-0665b42a/), and [Seokho Son](https://www.linkedin.com/in/seokho-son/).